MakeHist/mumu.h

```diff
@@ -466,18 +472,18 @@ mumu::mumu(TTree *tree) : fChain(0)
       dir->GetObject("t",tree);
 */
 #ifdef SINGLE_TRmumu
-       TFile *f = (TFile*)gROOT->GetListOfFiles()->FindObject("/xrootd/store/user/hyeahyun/TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8/crab_20180719_234954/180719_145025/0000/");
+       TFile *f = (TFile*)gROOT->GetListOfFiles()->FindObject("/xrootd/store/user/hyeahyun/TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8/crab_20180802_173830/180802_083858/0000/");
```

Comment:
why is this hardcoded and not passed as a parameter? Same comment for other instances